File: sys-config/Model/CMA/LOV/IdentityType.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace SystemConfigurator.Model.CMA.LOV;

[Table("cma_tbl_lov_customer_identity_type")]
public class IdentityType
{
    [Key]
    [Column("identity_type_id")]
    [StringLength(20)]
    public string IdentityTypeId { get; set; } = "";

    [Required]
    [Column("created_at")]
    public DateTime CreatedAt { get; set; } = DateTime.UtcNow;

    [Column("updated_at")]
    public DateTime? UpdatedAt { get; set; }

    [Column("deleted_at")]
    public DateTime? DeletedAt { get; set; }

    // Navigation Property
    public ICollection<Customer> Customers { get; set; } = new List<Customer>();
}

File: kyc-service/src/main/java/com/devcraftlabs/kyc/model/entity/lov/IdentityType.java
package com.devcraftlabs.kyc.model.entity.lov;

import java.time.Instant;

import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.Id;
import jakarta.persistence.Table;
import lombok.Data;

@Entity
@Table(name = "cma_tbl_lov_customer_identity_type")
@Data
public class IdentityType {

    @Id
    @Column(name = "identity_type_id", nullable = false, length = 20)
    private String identityTypeId;

    @Column(name = "created_at", nullable = false)
    private Instant createdAt = Instant.now();

    @Column(name = "updated_at")
    private Instant updatedAt;

    @Column(name = "deleted_at")
    private Instant deletedAt;
}